### DISCUSSION TITLE
Eslint Configuration

### DISCUSSION THREAD

**Discussion by mbuguanewton** [jaredpalmer/razzle#1624] [Q&A] [answered]
I am trying to make eslint work on vscode but it doesn't seem to. Is there a specific way to setup eslint on razzle? Can you set it up like in CRA?

**Comment by fivethreeo** [comment, marked as answer]
There is a very undocumented plugin https://github.com/jaredpalmer/razzle/tree/master/packages/razzle-plugin-eslint . ;)

**Reply by feruzm** [reply]
@newtfrank Have you tried it out, do you have any examples to share?

**Reply by mbuguanewton** [reply]
@feruzm  yes I did try it out. The plugin works though it has warnings on not receiving any updates. I will create a better example and share the link.

**Reply by fivethreeo** [reply]
The “no updates” may be unrelated. May be just unnamed exports in modules
